roles/serverspec/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: install ruby packages
+  apt: pkg={{ item }}
+  with_items:
+    - ruby1.9.1-full
+
+- name: install ruby gems
+  gem: name={{ item }} user_install=no
+  with_items:
+    - rake
+    - rspec-core
+
+- name: install net-ssh gem
+  gem: name=net-ssh version=2.9.2
+       user_install=no
+
+- name: install serverspec gem
+  gem: name=serverspec version=2.22.0
+       user_install=yes
+
+- name: install serverspec-extended-types gem
+  gem: name=serverspec-extended-types version=0.0.3
+       user_install=no
+
+- name: ensure serverspec directory exists
+  file: dest=/etc/serverspec/spec/localhost state=directory
+        owner=root mode=0755 recurse=true
+
+- name: serverspec rakefile
+  template: src=etc/serverspec/Rakefile
+            dest=/etc/serverspec/Rakefile mode=0755
+
+- name: serverspec helper
+  template: src=etc/serverspec/spec/spec_helper.rb
+            dest=/etc/serverspec/spec/spec_helper.rb mode=0755
+
+- name: monitoring common serverspec checks
+  template: src={{ item }}
+            dest=/etc/serverspec/spec/localhost/
+            mode=0755
+  with_fileglob: ../templates/etc/serverspec/spec/localhost/*
```

Comment:
Why would monitoring common serverspec checks go here, instead of in the monitoring-common role?